requirements.txt

```diff
@@ -1,13 +1,2 @@
-Django==1.11.12
-django_extensions==2.0.7
-django-user-agents==0.3.2
-MySQL-python==1.2.5
 nose==1.3.7
 pylint==1.8.4
```

Comment:
Is there any reason these requirements can't be satisfied by PIP?

Reply:
The reason these are separate is because they are not required for project installation (they are only used in testing) as such I think it would be the wrong move to install them as default with the package. However, with the enhancements to setup.py coming soon in #70 it is possible we can move these to be installed with a developer installation.